Furball.Vixie/Graphics/Backends/Veldrid/Abstractions/TextureRenderTargetVeldrid.cs

```diff
@@ -0,0 +1,66 @@
+using System.Numerics;
+using Veldrid;
+
+namespace Furball.Vixie.Graphics.Backends.Veldrid.Abstractions {
+    public sealed class TextureRenderTargetVeldrid : TextureRenderTarget {
+        private readonly VeldridBackend _backend;
+        public override Vector2 Size {
+            get;
+            protected set;
+        }
+
+        private TextureVeldrid _tex;
+        private Framebuffer    _fb;
+        
+        public TextureRenderTargetVeldrid(VeldridBackend backend, uint width, uint height) {
+            this._backend = backend;
+
+            this.Size = new(width, height);
+
+            this._tex = new TextureVeldrid(backend, width, height);
+            
+            FramebufferDescription description = new() {
+                ColorTargets = new[] {
+                    new FramebufferAttachmentDescription(this._tex.Texture, 0)
+                }
+            };
+
+            this._fb = backend.ResourceFactory.CreateFramebuffer(description);
+
+            if (!this._backend.GraphicsDevice.IsUvOriginTopLeft)
+                this._tex.IsFbAndShouldFlip = true;
+        }
+        
+        public override void Bind() {
+            this._backend.Flush();
+            
+            this._backend.CommandList.SetFramebuffer(_fb);
+            this._backend.CommandList.SetFullViewports();
+            
+            this._backend.SetProjectionMatrix(this._fb.Width, this._fb.Height);
+        }
+        
+        public override void Unbind() {
+            this._backend.Flush();
+            
+            this._backend.CommandList.SetFramebuffer(this._backend.RenderFramebuffer);
+            
+            this._backend.CommandList.SetFullViewports();
```

Comment:
if you set the proj matrix while binding, u should reset it when unbinding or no?